cmd/executor/cmd/root.go

```diff
@@ -227,6 +230,37 @@ func resolveSourceContext() error {
 	return nil
 }
 
+func resolveRelativePaths() error {
```

Comment:
@huguesalary Thank you for your contributions. This looks really good. However can we please some unit test for this?

Can you refactor `resolveRelativePaths()` to accept opts? That ways we can easily write unit tests


```suggestion
func resolveRelativePaths(opts config.KanikoOptions{}) error {
```

Now, you can creates tests where

```
func TestResolveRelativePaths(t *testing.T) {
	tests := []struct {
		description  string
		opts config.KanikoOptions
               expected config.KanikoOptions
               shdErr  bool 
	}{
            {
                description: "dockerfile path  is absolute",
                opts: config.KanikoOptions{
                           DockerfilePath: "/abs/dockerfile",
                        },
               expected: config.KanikoOptions{
                           DockerfilePath: "/abs/dockerfile",
                        },
            }

	}
	for _, tt := range tests {
           actual := resolveRelativePaths(tt.opts)
           t.CheckDeepEqualAndError(tt.shdErr, actual, tt.expected, tt.opts)
	}
}


```